mistletoe/core_tokens.py

```diff
@@ -1,18 +1,17 @@
 import re
-
+import unicodedata
 
 whitespace = {' ', '\t', '\n', '\x0b', '\x0c', '\r'}
-unicode_whitespace = {'\t', '\n', '\x0b', '\x0c', '\r', '\x1c', '\x1d', '\x1e',
```

Comment:
The number of items in this set is different from what is defined in the spec.

Due to sick of checking those non-readable characters one by one, I simply change it to a `is_unicode_whitespace` function.

See the comment on line 9 for more.